### ISSUE TITLE
Port various CAS types

### ISSUE THREAD

**Issue by ioanacrant** [dotnet/corefx#11115]
```
T:System.Security.SuppressUnmanagedCodeSecurityAttribute
M:System.Security.SuppressUnmanagedCodeSecurityAttribute.#ctor
```

```
T:System.Security.SecurityTreatAsSafeAttribute
M:System.Security.SecurityTreatAsSafeAttribute.#ctor
```
The above two have usage and should be ported.

```
P:System.Security.AllowPartiallyTrustedCallersAttribute.PartialTrustVisibilityLevel
M:System.Security.AllowPartiallyTrustedCallersAttribute.get_PartialTrustVisibilityLevel
M:System.Security.AllowPartiallyTrustedCallersAttribute.set_PartialTrustVisibilityLevel(System.Security.PartialTrustVisibilityLevel)
```

```
T:System.Security.PartialTrustVisibilityLevel
F:System.Security.PartialTrustVisibilityLevel.NotVisibleByDefault
F:System.Security.PartialTrustVisibilityLevel.value__
F:System.Security.PartialTrustVisibilityLevel.VisibleToAllHosts
```

```
T:System.Security.Authentication.ExtendedProtection.ExtendedProtectionPolicyTypeConverter
M:System.Security.Authentication.ExtendedProtection.ExtendedProtectionPolicyTypeConverter.#ctor
```

```
T:System.Security.SecurityRulesAttribute
M:System.Security.SecurityRulesAttribute.#ctor(System.Security.SecurityRuleSet)
P:System.Security.SecurityRulesAttribute.RuleSet
M:System.Security.SecurityRulesAttribute.get_RuleSet
P:System.Security.SecurityRulesAttribute.SkipVerificationInFullTrust
M:System.Security.SecurityRulesAttribute.get_SkipVerificationInFullTrust
M:System.Security.SecurityRulesAttribute.set_SkipVerificationInFullTrust(System.Boolean)
```

```
T:System.Security.SecurityRuleSet
F:System.Security.SecurityRuleSet.Level1
F:System.Security.SecurityRuleSet.Level2
F:System.Security.SecurityRuleSet.None
F:System.Security.SecurityRuleSet.value__
```

**Comment by danmosemsft** [comment]
Blocked on CoreCLR update.

**Comment by danmosemsft** [comment]
These are still in .NET Standard.

**Comment by danmosemsft** [comment]
Fixed with 9b270c7b06751ab24d98d1404e4fd671c63aeb9e
